### PR TITLE
Feature/mn 256/update damage pop up

### PR DESCRIPTION
**Pull request by rohitsharma120582**
Implementing the small popup that pops up when the user wants to add or edit a damage on the car.

## Technical description
- Create a component called `UpdateDamagePopUp`
- Add animation to show the component from bottom
- Add gesture on small-bar to slide the popup from "minimal" to "full" and vice versa

## Pending
- Add translations for car parts
- Integrating `DamageManipulator` component

## UI

<p>
Screenshot 2023-04-26 at 1 33 42 PM
Screenshot 2023-04-26 at 1 33 55 PM
</p>

### CODE REVIEW

File: packages/inspection-report/src/components/DamageReport/DamageReport.js [hunk 1]
```diff
@@ -51,16 +54,23 @@ const styles = StyleSheet.create({
 function DamageReport() {
   const { t } = useTranslation();
   const [tabIndex, setTabIndex] = useState(0);
+  const [isPopVisible, setIsPopVisible] = useState(false);
```
Comment (souyahia-monk) [inline packages/inspection-report/src/components/DamageReport/DamageReport.js:57]: rename to `isPopUpVisible, setIsPopUpVisible`

File: packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+    if (viewMode === 'full' && gestureState.dy >= 0) {
+      setViewMode('minimal');
+    } else if (viewMode === 'minimal') {
+      if (gestureState.dy <= 0) {
+        setViewMode('full');
+      } else {
+        scrollOut();
+      }
+    }
+  };
```
Comment (souyahia-monk) [inline packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js:126]: define these callbacks with `useCallback`

**Comment by souyahia-monk** [comment]
Don't forget to add translations everywhere in your component! I will add the french translations later

File: packages/inspection-report/src/components/DamageReport/DamageReport.js (hunk 1 above)
Comment (rohitsharma120582) [reply packages/inspection-report/src/components/DamageReport/DamageReport.js]: Done!

File: packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js (hunk 2 above)
Comment (rohitsharma120582) [reply packages/inspection-report/src/components/DamageReport/UpdateDamagePopUp/index.js]: Done

**Comment by rohitsharma120582** [comment]
> Don't forget to add translations everywhere in your component! I will add the french translations later

Yes, working on it...

**Pull request closed without merging**